wp-html-mail/template.html

```diff
@@ -0,0 +1,183 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta http-equiv="content-language" content="en">
+		<title>NHS Leadership Academy</title>
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+        <style type="text/css">
+			body{width:100% !important; -webkit-text-size-adjust:none; margin:0; padding:0;}
+			.ReadMsgBody{width:100%;}
```

Comment:
This selector class `ReadMsgBody` is never used in the following code, same `ExternalClass` below

Reply:
All unused classes are now gone